shortcut_composer/api_krita/enums/action.py

```diff
@@ -0,0 +1,581 @@
+    STYLE_MENU = "style_menu" # Styles
+    WINDOWS_CASCADE = "windows_cascade" # Cascade
+    WINDOWS_TILE = "windows_tile" # Tile
+    WINDOWS_NEXT = "windows_next" # Next
+    WINDOWS_PREVIOUS = "windows_previous" # Previous
+    VIEW_NEWWINDOW = "view_newwindow" # New Window
+    FILE_CLOSE = "file_close" # Close
+    FILE_SESSIONS = "file_sessions" # Sessions...
+    COMMAND_BAR_OPEN = "command_bar_open" # Search Actions
+    OPTIONS_CONFIGURE = "options_configure" # Configure Krita...
+    EXPANDING_SPACER_0 = "expanding_spacer_0" # Expanding Spacer
+    EXPANDING_SPACER_1 = "expanding_spacer_1" # Expanding Spacer
+    HELP_CONTENTS = "help_contents" # Krita Handbook
+    HELP_REPORT_BUG = "help_report_bug" # Report Bug...
+    SWITCH_APPLICATION_LANGUAGE = "switch_application_language" # Switch Application Language...
+    HELP_ABOUT_APP = "help_about_app" # About Krita
+    HELP_ABOUT_KDE = "help_about_kde" # About KDE
+    RECORDER_RECORD_TOGGLE = "recorder_record_toggle" # Record Timelapse
+    RECORDER_EXPORT = "recorder_export" # Export Timelapse...
+    MAIN_TOOL_BAR = "mainToolBar" # Hide File Toolbar
+    BRUSHES_AND_STUFF = "BrushesAndStuff" # Hide Brushes and Stuff Toolbar
+
+    # Built-in Scripts (The ones that make sesne to include)
+    COLOR_SPACE = "color_space" # Colour Space
+    DOCUMENT_TOOLS = "document_tools" # Document Tools
+    EXPORT_LAYERS = "export_layers" # Export Layers
+    FILTER_MANAGER = "filter_manager" # Filter Manager
+    PLUGIN_IMPORTER_FILE = "plugin_importer_file" # Import Python Plugin from File...
+    PLUGIN_IMPORTER_WEB = "plugin_importer_web" # Import Python Plugin from Web...
+    PYTHON_SCRIPTER = "python_scripter" # Scripter
+    TEN_SCRIPTS = "ten_scripts" # Ten Scripts
+    EXECUTE_SCRIPT_1 = "execute_script_1" # Execute Script 1
+    EXECUTE_SCRIPT_2 = "execute_script_2" # Execute Script 2
+    EXECUTE_SCRIPT_3 = "execute_script_3" # Execute Script 3
+    EXECUTE_SCRIPT_4 = "execute_script_4" # Execute Script 4
+    EXECUTE_SCRIPT_5 = "execute_script_5" # Execute Script 5
+    EXECUTE_SCRIPT_6 = "execute_script_6" # Execute Script 6
+    EXECUTE_SCRIPT_7 = "execute_script_7" # Execute Script 7
+    EXECUTE_SCRIPT_8 = "execute_script_8" # Execute Script 8
+    EXECUTE_SCRIPT_9 = "execute_script_9" # Execute Script 9
+    EXECUTE_SCRIPT_10 = "execute_script_10" # Execute Script 10
+
+    # ShortcutComposer actions (The ones that make sense to use this way)
+    RELOAD_SHORTCUT_COMPOSER = "Reload Shortcut Composer" # Reload Shortcut Composer
+    CONFIGURE_SHORTCUT_COMPOSER = "Configure Shortcut Composer" # Configure Shortcut Composer
+
+    def activate(self):
+        Api.instance().action(self.value).trigger()
+
+    @property
+    def icon(self) -> QIcon:
+        """Return the icon of this tool."""
+        icon_name = _ICON_NAME_MAP.get(self, "edit-delete")
+        return Api.instance().icon(icon_name)
+
+
+_ICON_NAME_MAP = {
+}
+"""Maps actions to names of their icons."""
```

Comment:
Nie testowałem tego jeszcze, ale jest szansa, że `Krita.instance().action("nazwa").icon` zwróci ikonkę akcji. Wtedy nie trzeba by robić żadnych takich mapowań i wywalić je w kontrolerze Tool (przez co nigdy nie będzie już tam niezgodności)

Reply:
Roger, zrobię to i przetestuje czy działa.